### ISSUE TITLE
Enhancement request: option to recognize apostrophe as a word boundary

### ISSUE THREAD

**Issue by BenHocking** [streetsidesoftware/vscode-spell-checker#531]
I'm loving the extension and all of the ways you can configure it. However, in Ada, [apostrophes are used as a delimiter](https://en.wikibooks.org/wiki/Ada_Programming/Delimiters/%27) to specify type attributes. Code Spell Checker is probably not familiar with Ada  (understandable), so it appears to be assuming that an apostrophe between two letters is being used in the usual sense as a possessive or as a contraction. This is causing things like `vertices'First`, etc. to show up as spelling errors.

Feature enhancement: allow the user to specify (either globally in the JSON file or per file in the comments) symbols that should be recognized as delimiters. Optionally, just check to see if strings on both sides of an apostrophe are words and if so, don't flag `word1'word2` as a non-word.

**Comment by Jason3S** [comment]
It is already possible to do this by telling the spell checker to ignore the `'`.

In VS Code settings:
```json
    "cSpell.languageSettings": [
    {
        "languageId": "ada",
        "patterns": [{
            "name": "ada word break",
            "pattern": "/(?<=\\w)['](?=\\w)/g",
            "description": "Ada Word Break"
        }],
        "ignoreRegExpList": ["ada word break"]
    }
    ]
```

In `cspell.json`:
```json
    "languageSettings": [
    {
        "languageId": "ada",
        "patterns": [{
            "name": "ada word break",
            "pattern": "/(?<=\\w)['](?=\\w)/g",
            "description": "Ada Word Break"
        }],
        "ignoreRegExpList": ["ada word break"]
    }
    ]
```

**Comment by BenHocking** [comment]
Awesome, thanks!
